tests/ext/sandbox/deferred_load_missing_load_fn.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+deferred loading doesn't trigger nor crash if DDTrace\Integrations\load_deferred_integration is missing
```

Comment:
Would running a variant of the `distributed_tracing_curl.phpt` test without including the [fake injector](https://github.com/DataDog/dd-trace-php/blob/master/tests/ext/sandbox/distributed_tracing_curl.phpt#L8) test for a similar class of issues?

Reply:
Theoretically, yes it would!